ui/packages/ui/src/main/webapp/js/models/SessionTimeout.js

```diff
@@ -73,19 +105,22 @@ define([
                 clearTimeout(this.logoutTimer);
             },
             resetIdleTimeoutDate: function () {
-                this.set('idleTimeoutDate', getIdleTimeoutDate());
+                var idleTimeoutDate = getIdleTimeoutDate();
+                localStorage.setItem('idleTimeoutDate', idleTimeoutDate);
+                this.set('idleTimeoutDate', idleTimeoutDate);
             },
             startListeningForUserActivity: function () {
-                $(document).on('keydown.sessionTimeout mousedown.sessionTimeout mousemove.sessionTimeout', _.throttle(this.resetIdleTimeoutDate.bind(this), 5000));
```

Comment:
Did we remove the mousemove event for performance reasons?

Reply:
That shouldn't be necessary, that's why we throttled it.

Reply:
Mainly it was for convenience during development; I kept refreshing the session on accident by bumping the mouse. I do think the mousemove is a bit aggressive, though.

I don't have a strong opinion. I'll change it back if that's preferred

Reply:
I think a user might have a different definition of convenience. 😃 If you're referring to the throttling, I'm fine with upping it if you prefer. 5 seconds shouldn't have been causing any performance impact though.

Reply:
Imo button presses and mouse movement are stronger indications of purposeful interaction with a page than mouse movement. Tracking mouse movement, we might register activity that a user isn't even aware of, and then it's possible that the user gets confused because the prompts don't behave like they expected. That's what I meant by too aggressive.